sentry_sdk/integrations/wsgi.py

```diff
@@ -82,8 +82,8 @@ def __call__(self, environ, start_response):
         _wsgi_middleware_applied.set(True)
         try:
             hub = Hub(Hub.current)
-            with auto_session_tracking(hub, session_mode="request"):
-                with hub:
+            with hub:
+                with auto_session_tracking(hub, session_mode="request"):
```

Comment:
Why are we still using the `Hub` here?

Reply:
The hub is still used in a lot of places. Under the hood the new scopes api is used. We will update all integrations bit by bit to use the new api.